packages/codemods/transforms/v3/dropdown/__testfixtures__/basic.input.hbs

```diff
@@ -0,0 +1,10 @@
+<Hds::Dropdown @listPosition="left" as |dd|>
```

Comment:
I would probably use a much simpler code snippet, with a single child (unless we're worried it may modify the children too, in which case I would add one per kind of item)

Reply:
yes, that works for me, just wanted to make sure we have node elements in the block content